server.spec.js

```diff
@@ -0,0 +1,107 @@
+//import server 
+const server = require('./api/server.js')
+
```

Comment:
Great job taking on the challenge of adding tests to your project Christopher 👍 , testing your code regularly is a great way to set yourself apart in the job market.